.github/workflows/reusable-appetize.yml

```diff
@@ -31,7 +33,9 @@ jobs:
       uses: ./.github/workflows/reusable-android-build.yml
       secrets: inherit
       with:
-        branch: ${{ github.base_ref }} 
+        branch: ${{ github.ref }} 
```

Comment:
The reusable branch input refers to the branch of the deployment repo to checkout, not the ref of the builder repo which is set via `env.APP_CODE_BRANCH`

So I don't think it's possible to get the reusable action working in this way when being called from the builder repo itself. Instead we would need to add a different action for the builder repo like we do for pr-preview